test/e2e/usersignup_test.go

```diff
@@ -189,82 +165,29 @@ func (s *userSignupIntegrationTest) checkUserSignupManualApproval() {
 	})
 }
 
-func (s *userSignupIntegrationTest) assertCreatedMUR(userSignup *v1alpha1.UserSignup) *v1alpha1.MasterUserRecord {
-	mur, err := s.hostAwait.WaitForMasterUserRecord(userSignup.Status.CompliantUsername)
-	require.NoError(s.T(), err)
-
-	require.Len(s.T(), mur.Spec.UserAccounts, 1)
-	assert.Equal(s.T(), userSignup.Name, mur.Labels["toolchain.dev.openshift.com/user-id"])
-	assert.Equal(s.T(), userSignup.Name, mur.Spec.UserID)
-	assert.Equal(s.T(), "default", mur.Spec.UserAccounts[0].Spec.NSLimit)
-	assert.NotNil(s.T(), mur.Spec.UserAccounts[0].Spec.NSTemplateSet)
-	if userSignup.Spec.TargetCluster != "" {
-		// Target cluster set manually from spec
-		assert.Equal(s.T(), userSignup.Spec.TargetCluster, mur.Spec.UserAccounts[0].TargetCluster)
-	} else {
-		// Target cluster set automatically
-		assert.NotEmpty(s.T(), mur.Spec.UserAccounts[0].TargetCluster)
-	}
-
-	return mur
-}
-
 func newUserSignup(t *testing.T, host *wait.HostAwaitility, username string, email string) *v1alpha1.UserSignup {
 	memberCluster, ok, err := host.GetKubeFedCluster(cluster.Member, wait.ReadyKubeFedCluster)
 	require.NoError(t, err)
 	require.True(t, ok)
 
+	md5hash := md5.New()
+	_, _ = md5hash.Write([]byte(email))
+	emailHash := hex.EncodeToString(md5hash.Sum(nil))
+
```

Comment:
could we have one unified place where we create all `UserSignup` instances? ideally using the same way as is used from the registration-service: https://github.com/codeready-toolchain/toolchain-e2e/blob/master/test/e2e/e2e_test.go#L255-L271

Reply:
You want to create all `UserSignup` instances via the registration service API?  We can't actually do this for the banned user test where the user is banned prior to user registration, as the registration service now queries for any matching `BannedUser` resources before allowing the registration to proceed.  This would break some of our tests.  It seems to me that `newUserSignup` is already used extensively for creating `UserSignup` instances.  In fact, after checking all of `base_user_test` and `user_management_test` code it seems this is the *only* place where the instance is created.

Reply:
I already mentioned that it's possible covering the other cases too, but keep it as it is :+1: